test/box-luatest/gh_7000_compat_module_test.lua

```diff
@@ -0,0 +1,270 @@
+local t = require('luatest')
+local g = t.group()
+local compat = require('compat')
+
+local option_1_called = false
+local option_2_called = false
+
+local policies = {
+  option_1_policy = {'option_1',
+    {old = false,
+     new = true,
+     default = 'new',
+     brief = "option_1",
+     doc = "https://tarantool.io",
+     frozen = false
+    },
+    function()
+        option_1_called = true
+    end
+  },
+  option_2_policy = {'option_2',
+    {old = true,
+     new = false,
+     default = 'old',
+     brief = "option_2",
+     doc = "https://tarantool.io",
+     frozen = false
+    },
+    function()
+        option_2_called = true
+    end
+  },
+}
+
+local frozen_policy = {'frozen_option',
+    {old = true,
+     new = false,
+     default = false,
+     brief = "frozen_option",
+     doc = "https://tarantool.io",
+     frozen = true
+    },
+    nil
+  }
+
+local test_options_calls = function()
+    t.assert(option_1_called)
+    t.assert(option_2_called)
+    option_1_called = false
+    option_2_called = false
+end
+
+g.test_add_options = function()
+    for _, elem in pairs(policies) do
+        compat.add_options({elem})
+    end
+    t.assert_equals(type(compat.option_1), "table")
+    t.assert_equals(type(compat.option_2), "table")
+    test_options_calls()
+
+    local bad_policy = {'bad_option_1',                -- frozen option with default == old
+                        {old = true,
+                         new = false,
+                         default = true,
+                         frozen = true,
+                         brief = 'bad_option',
+                         doc = 'https://tarantool.io',
+                        }, nil}
+
+    t.assert_error(compat.add_options, bad_policy)
+
+    bad_policy = {'bad_option_2',                      -- unfrozen option with bad action
```

Comment:
All comments started with uppercase letter and finished with dot. Please fix in a whole patch.

Reply:
Done